src/pages/developing/web-components-tutorial/step-2.mdx

```diff
@@ -0,0 +1,140 @@
+  src=""
+  frameborder="no"
+  allowtransparency="true"
+  allowfullscreen="true"
+  className="bx--iframe bx--iframe--border"
+/>
+
+## Fork, clone and branch
+
+This tutorial has an accompanying GitHub repository called
+[carbon-for-ibm-dotcom-tutorial](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-tutorial)
+that we'll use as a starting point for each step. If you haven't forked and
+cloned that repository yet, and haven't added the upstream remote, go ahead and
+do so by following the <Link to="/developing/web-components-tutorial">step 1
+instructions</Link>.
+
+### Branch
+
+With your repository all set up, let's check out the branch for this tutorial
+step's starting point.
+
+```bash
+git fetch upstream
+git checkout -b web-components-step-2 upstream/web-components-step-2
+```
+
+<InlineNotification>
+
+**Note:** This builds on top of step 1, but be sure to check out the upstream
+step 2 branch because it includes the static assets required to get through this
+step.
+
+</InlineNotification>
+
+TBD
+
+## Submit pull request
+
+We're going to submit a pull request to verify completion of this tutorial step.
+
+### Continuous integration (CI) check
+
+Run the CI check to make sure we're all set to submit a pull request.
+
+```bash
+yarn ci-check
+```
+
+<InlineNotification>
+
+**Note:** Having issues running the CI check?
+
+<Link to="/developing/web-components-tutorial/step-1#continuous-integration-(ci)-check">
+  Step 1
+</Link>
+has troubleshooting notes that may help.
+
+</InlineNotification>
+
+### Git commit and push
+
+Before we can create a pull request, stage and commit all of your changes:
+
+```bash
+git add --all && git commit -m "feat(tutorial): complete step 2"
+```
+
+Then, push to your repository:
+
+```bash
+git push origin web-components-step-2
+```
+
+<InlineNotification>
+
+**Note:** Having issues pushing your changes?
+
+<Link to="/developing/web-components-tutorial/step-1#git-commit-and-push">
+  Step 1
+</Link>
+has troubleshooting notes that may help.
+
+</InlineNotification>
+
+### Pull request (PR)
+
+Finally, visit
+[carbon-for-ibm-dotcom-tutorial](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-tutorial)
+to "Compare & pull request". In doing so, make sure that you are comparing to
+`web-components-step-2` into `base: web-components-step-2`.
+
+<InlineNotification>
+
+**Note:** Expect your tutorial step PRs to be reviewed by the Carbon for IBM.com
+team but not merged. We'll close your PR so we can keep the repository's remote
+branches pristine and ready for the next person!
+
+</InlineNotification>
+
+<InlineNotification>
+
+**Note:** If your PR fails the Github Actions test with the error
+`Can't make a request in offline mode`, try running the following command:
+`rm -rf .yarn-offline-mirror node_modules && yarn cache clean && yarn install`.
+Add and commit the changes once this completes, and try pushing again.
+
+</InlineNotification>
```

Comment:
It would be good to avoid the doubled up notifications. I don't think you need them for either of them sentences.